novawallet/Common/Services/ChainRegistry/ChainRegistry.swift

```diff
@@ -351,7 +351,7 @@ extension ChainRegistry: ChainRegistryProtocol {
     }
 
     func unsubscribeChainState(_ subscriber: ConnectionStateSubscription, chainId: ChainModel.Id) {
-        connectionPool.subscribe(subscriber, chainId: chainId)
+        connectionPool.unsubscribe(subscriber, chainId: chainId)
```

Comment:
btw seems like we there was a small mistake that prevented us from unsubscribing from the chain state

Reply:
Yeah, my bad